js/data.js

```diff
@@ -1048,17 +1125,39 @@ var data = {
         dozent: ["Dr. Matthias Uflacker", "Dipl.-Kfm. Jens Krüger", "Thomas Kowark", "Franziska Häger"],
         nameLV: "Global Team-Based Product Innovation & Engineering I - ME 310"
     },
+    productInnovationEngineeringI_HCT: {
+        kurz: "ME310 I (mit HCT)",
+        kennung: [ITSE, BPET, HCT],
```

Comment:
Can't find this with `HCT` in the current version: https://hpi.de/fileadmin/user_upload/hpi/dokumente/studiendokumente/master/stundenplaene/ws_201516/lv_angebot_master_ws201516.pdf.